MoltenVK/MoltenVK/GPUObjects/MVKDevice.h

```diff
@@ -869,6 +870,8 @@ class MVKDevice : public MVKDispatchableVulkanAPIObject {
 	id<MTLSamplerState> _defaultMTLSamplerState = nil;
 	id<MTLBuffer> _dummyBlitMTLBuffer = nil;
     uint32_t _globalVisibilityQueryCount = 0;
+	int _capturePipeFildes = -1;
+	std::string _capturePipeFileName;
```

Comment:
Where possible, we do try to organize to minimize dead space in the underlying class structures. Recommend moving `_capturePipeFileName` up a few lines to below `std::mutex`.

And it took me a while to mentally parse `Fildes`! 😉 

I'll suggest renaming `_capturePipeFildes` to `_capturePipeFileDesc`, to match pattern in `_capturePipeFileName`.

Reply:
> And it took me a while to mentally parse `Fildes`! 😉

Yeah. That's actually a somewhat common abbreviation you see in Unix manpages. The name of a file descriptor parameter is often `fildes`.

Personally, I prefer `_capturePipeFD`. It's shorter, and the initialism "FD" is well known.

Reply:
I went with FileDesc, feel free to change.